examples/evaluation_and_profiling/swe_bench/src/nat_swe_bench/config.py

```
@@ -35,12 +46,12 @@ class SweBenchPredictorSkeletonConfig(SweBenchPredictorBaseConfig, name="skeleto
 
-SweBenchPredictorConfig = typing.Annotated[typing.Annotated[SweBenchPredictorGoldConfig,
-                                                            Tag(SweBenchPredictorGoldConfig.static_type())]
-                                           | typing.Annotated[SweBenchPredictorSkeletonConfig,
-                                                              Tag(SweBenchPredictorSkeletonConfig.static_type())],
-                                           Discriminator(TypedBaseModel.discriminator)]
-
+SweBenchPredictorConfig = typing.Annotated[
+    typing.Annotated[SweBenchPredictorFullConfig, Tag(SweBenchPredictorFullConfig.static_type())]
+    | typing.Annotated[SweBenchPredictorGoldConfig, Tag(SweBenchPredictorGoldConfig.static_type())]
+    | typing.Annotated[SweBenchPredictorSkeletonConfig, Tag(SweBenchPredictorSkeletonConfig.static_type())]
+    | typing.Annotated[SweBenchPredictorIterativeConfig, Tag(SweBenchPredictorIterativeConfig.static_type())],
```

Ruff (0.14.11)

Undefined name `SweBenchPredictorIterativeConfig`

(F821)